praw/models/reddit/live.py

```diff
@@ -37,3 +59,96 @@ def __init__(self, reddit, id=None,  # pylint: disable=redefined-builtin
+        path = API_PATH['live_updates'].format(id=self.id)
+        for update in ListingGenerator(self._reddit, path,
+                                       **generator_kwargs):
+            update.thread_id = self.id
+            yield update
+
+
+class LiveUpdate(RedditBase):
+    """An individual :class:`.LiveUpdate` object."""
+
+    STR_FIELD = 'id'
+
+    def __init__(self, reddit, thread_id=None, update_id=None, _data=None):
+        """Initialize a lazy :class:`.LiveUpdate` instance.
+
+        Either ``thread_id`` and ``update_id``, or ``_data`` must be
+        provided.
+
+        .. warning:: In the current implementation, accessing lazy attributes
+           (e.g., ``update.author``) is very slow.
+
+        :param reddit: An instance of :class:`.Reddit`.
+        :param thread_id: A live thread ID, e.g., ``'ukaeu1ik4sw5'``.
+        :param update_id: A live update ID, e.g.,
+            ``'7827987a-c998-11e4-a0b9-22000b6a88d2'``.
+
+        .. code-block:: python
+
+           update = LiveUpdate(reddit, 'ukaeu1ik4sw5',
+                               '7827987a-c998-11e4-a0b9-22000b6a88d2')
+           update.thread_id  # 'ukaeu1ik4sw5'
+           update.id         # '7827987a-c998-11e4-a0b9-22000b6a88d2'
+           update.author     # 'wting'
+        """
+        if thread_id and update_id and not _data:
+            super(LiveUpdate, self).__init__(reddit, _data)
+            self.thread_id = thread_id
+            self.id = update_id  # pylint: disable=invalid-name
+        elif (not thread_id and not update_id) and _data:
+            super(LiveUpdate, self).__init__(reddit, _data)
+            # `thread_id` should be set until users access that.
+            # See `LiveThread.updates` and `LiveThread.__getitem__`.
+        else:
+            raise TypeError('Either `thread_id` and `update_id`, or '
+                            '`_data` must be provided.')
+
+    def __getattr__(self, attribute):
+        """Return specified attribute which is not found.
+
+        Raise PRAWException when specified attribute is ``thread_id``.
+        """
+        if attribute == 'thread_id':
+            raise PRAWException('Can\'t set `thread_id` by _fetch()')
+        return super(LiveUpdate, self).__getattr__(attribute)
+
+    def __setattr__(self, attribute, value):
+        """Objectify author."""
+        if attribute == 'author':
+            value = Redditor(self._reddit, name=value)
+        super(LiveUpdate, self).__setattr__(attribute, value)
+
+    def _fetch(self):
+        thread = LiveThread(self._reddit, self.thread_id)
+        for update in thread.updates(limit=None):
+            if update.id == self.id:
+                self.__dict__.update(update.__dict__)
+                self._fetched = True
+                return
+        raise PRAWException('No {!r} data returned for {}'
+                            .format(self.__class__.__name__,
+                                    self.id))
+
+    def _info_path(self):
```

Comment:
Maybe I misunderstood about info path. I had thought the path is only used in _fetch but it was wrong.